### ISSUE TITLE
Support for ppc64le

### ISSUE THREAD

**Issue by YiannisGkoufas** [kahing/goofys#357]
Hi there,

I tried to use the library for ppc64le and I got the following error:

panic: Page size is unexpectedly 65536

```
goroutine 1 [running]:
github.com/jacobsa/fuse/internal/buffer.init.0()
	/root/gopath/src/github.com/jacobsa/fuse/internal/buffer/in_message.go:33 +0xe4
```

I am assuming it has to do with the underlying library that's why I opened an issue also there ( https://github.com/jacobsa/fuse/issues/44 )
I don't think we can do much unless it's fixed in the fuse library correct?

Thanks!

**Comment by kahing** [comment]
right. Since so few people have access to a PPC machine you may have to poke at this yourself
